test/linter/test-versions.js

```diff
@@ -49,16 +49,48 @@ for (const browser of Object.keys(browsers)) {
   }
 }
 
+/** @type {string[]} */
+const blockMany = [
+  'chrome',
+  'chrome_android',
+  'edge',
+  'firefox',
+  'firefox_android',
+  'ie',
+  'opera',
+  'opera_android',
+  'safari',
+  'safari_ios',
+  'samsunginternet_android',
+  'webview_android',
+];
+
+/** @type {object.<string, string[]>} */
+const blockList = {
```

Comment:
What do `blockMany` and `blockList` actually represent? I'm not really sure.

Reply:
`blockMany` is the list of all the browsers that we wish to have real values for, and `blockList` is the list of the browsers we require real values for per category.  I'm down for renaming them in a future PR to better clarify this!

Reply:
Thanks! Renaming is a good idea, and maybe a JSDoc could be helpful.

Reply:
Opened #16350 for that!